packages/app/src/TeamPage.tsx

```diff
@@ -742,6 +764,50 @@ function CreateWebhookForm({
           error={form.formState.errors.description?.message}
           {...form.register('description')}
         />
+        {form.getValues('service') === 'generic' && [
+          <label className=".mantine-TextInput-label" key="1">
+            Webhook Body (optional)
+          </label>,
+          <div className="mb-2" key="2">
+            <CodeMirror
+              height="100px"
+              extensions={[
+                json(),
+                linter(jsonLinterWithEmptyCheck()),
+                placeholder(
+                  '{\n\t"text": "{{title}} | {{body}} | {{link}}"\n}',
+                ),
+              ]}
+              theme="dark"
+              onChange={value => form.setValue('body', value)}
+            />
+          </div>,
+          <Alert
+            icon={<i className="bi bi-info-circle-fill text-slate-400" />}
+            key="3"
+            className="mb-4"
+            color="gray"
+          >
+            <span>
+              Currently the body supports the following message template
+              variables:
+            </span>
+            <br />
+            <span>
+              <code>
+                {'{{'}link{'}}'}
+              </code>
+              ,{' '}
+              <code>
+                {'{{'}title{'}}'}
+              </code>
+              ,{' '}
+              <code>
+                {'{{'}body{'}}'}
+              </code>
+            </span>
```

Comment:
style: it would be nice if we could reuse `DEFAULT_GENERIC_WEBHOOK_BODY` so the component matches it always